frontend/dockerfile/dockerfile_lint_test.go

```diff
@@ -489,6 +490,55 @@ COPY Dockerfile .
 			},
 		},
 	})
+
+	testUndeclaredArgEnv(t, sb)
+}
+
+func testUndeclaredArgEnv(t *testing.T, sb integration.Sandbox) {
```

Comment:
This should go to another PR. Make it a new test and don't combine with linter tests.

Reply:
Yeah, that definitely shouldn't be in here. Must have missed a `git checkout` at some point and not noticed.